src/main/java/teammates/common/datatransfer/attributes/GenderType.java

```diff
@@ -0,0 +1,18 @@
+package teammates.common.datatransfer.attributes;
+
+// CHECKSTYLE.OFF:import illegal package
+import com.google.appengine.repackaged.com.google.gson.annotations.SerializedName;
```

Comment:
Try to use import com.google.gson.annotations.SerializedName;
I believe in that case you will not need //CHECKSTYLE.OFF at line above.